### ISSUE TITLE
🐛 

### ISSUE THREAD

**Issue by relativemodder** [svelte-plugins/tooltips#41]
**Describe the bug**
Tried to make a tooltip from tutorial but...

![image](https://github.com/svelte-plugins/tooltips/assets/61388196/edc3534f-d2d3-473b-896a-3ef687a77f9d)

**To Reproduce**

```
import { tooltip } from "@svelte-plugins/tooltips";
    
<u title="Hello World!" use:tooltip={{ content: 'weiogjiowej' }}>
        <img class="brand-color-filter" alt={"Verified ass " + verify}
        src={verifyPaths.get(`${verify}`)} />
    </u>
```

**Expected behavior**
Working tooltip

**Desktop (please complete the following information):**

- Version: doesn't matter, it happens in vs code

**Comment by thibault60000** [comment]
Same issue :up:

**Comment by jgreywolf** [comment]
I think you need to at least have action prop as well.
`use:tooltip={{ content: 'weiogjiowej', action: 'hover' }}`

**Comment by dysfunc** [comment]
This should be resolved if you update to the latest release `3.0.1`
